main.html

```diff
@@ -1,47 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<meta charset="UTF-8">
-<title>Page Title</title>
-<link rel="stylesheet" href="main.css">
-<link rel="stylesheet" href="normalize.css">
+    <meta charset="UTF-8">
+    <meta content='width=device-width, initial-scale=1.0, user-scalable=0' name='viewport'/>
+    <title>Page Title</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css">
+    <link rel="stylesheet" href="main.css">
 </head>
 <body>
     <header>
         <div class="header">
             <figure class="header__image">
-                <img src = "lyft_logo.png" alt=""/>
+                <img src="lyft_logo.png" class ="header__image-source" alt="lyft-logo"/>
             </figure>
             <div class="header__small-container">
                 <div class="header__search">
-                    <textarea rows="3" cols="1" placeholder="Where are you go" class="header__textarea"></textarea>
+                    <textarea rows="3" cols="1" placeholder="Where are you going?" class="header__textarea"></textarea>
                     <div class="header__glass">
```

Comment:
it should be a label tag, and the end-user should be able to click on the grass and trigger input focus

Reply:
+